scripts/gen_chapters.js

```diff
@@ -0,0 +1,52 @@
+/* jshint node:true */
```

Comment:
We have read me for scripts. please add this scripts info also there

Reply:
ok I have added